lib/metadata/metadataUtils.js

```diff
@@ -84,6 +89,40 @@ function metadataGetObject(bucketName, objectKey, versionId, log, cb) {
         });
 }
 
+/** metadataGetObjects - retrieves specified object or version from metadata. This
+ * method uses cursors, hence is only compatible with a MongoDB DB backend.
+ * @param {string} bucketName - name of bucket
+ * @param {string} objectsKeys - name of object key
+ * @param {RequestLogger} log - request logger
+ * @param {function} cb - callback
+ * @return {undefined} - and call callback with err, bucket md and object md
+ */
+function metadataGetObjects(bucketName, objectsKeys, log, cb) {
+    const options = { getDeleteMarker: true };
+    const objects = objectsKeys.map(objectKey => ({
```

Comment:
there are already 2 maps in arsenal, and one more here : seems a lot, should maybe rework something to find a proper 'format' which suits all and avoids too many conversions...